determined-ai/determined
# chore: update license for web pacakges

## Description
[WEB-1145]

only added licenses of libraries in dependencies of package.json




## Test Plan
- Check if license of each package is correct
- Check if add/remove correct licenses (only dependencies in package.json)




## Commentary (optional)

- didnt add licenses for `notebook` and `@determined-ai/hermes-parallel-coordinates` because it doesnt have licenses
- ~~[eslint-plugin-sort-keys-fix](https://www.npmjs.com/package/eslint-plugin-sort-keys-fix) has license but doesnt have license in repo. what should we do in this case?~~
- ~~`npm-force-resolutions` is not prob used, but added the license because its in package.json~~





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1145]: https://hpe-aiatscale.atlassian.net/browse/WEB-1145?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
